_docs-as-code.rst

```diff
@@ -105,6 +100,22 @@ These could be PDFs or tarball downloads.
 This static content is stored in a ``_static/`` directory.
 So that static content from all packages can be integrated, each package must store static content in a sub-directory of the ``_static`` directory, such as ``_static/afw``.
 
+Data-only packages, like ``afwdata`` have package documentation directories named after the package/Git repository itself.
```

Comment:
I'm not sure these are always "data-only".  We could conceivably have a "C++-only" package, for example.  Comma after `afwdata`.